homeassistant/components/mystrom/light.py

```diff
@@ -135,7 +101,7 @@ async def async_turn_on(self, **kwargs: Any) -> None:
             color_h, color_s = 0, 0  # Back to white
 
         try:
-            if not self.is_on:
+            if not self._attr_is_on:
```

Comment:
Avoid reading the attribute. You should read via the property, and write via the attribute.
```suggestion
            if not self.is_on:
```